libraries/AP_RCTelemetry/AP_VideoTX.cpp

```diff
@@ -61,9 +61,16 @@ const AP_Param::GroupInfo AP_VideoTX::var_info[] = {
     // @DisplayName: Video Transmitter Options
     // @Description: Video Transmitter Options.
     // @User: Advanced
-    // @Bitmask: 0:Pitmode
+    // @Bitmask: 0:Pitmode disabled,1: PitMode enabled
```

Comment:
this is wrong. This is a bitmask so value 0 is disabled, value 1 (bit 0) is enabled

Reply:
So in order to map options : // @Bitmask: 0:Disabled,1: PitMode Enabled, 2:Unlocked

Reply:
So in order to map options : // @Bitmask: 0:Disabled,1: PitMode Enabled, 2:Unlocked

Reply:
Fixed, aditional param for locking, init and additional method removed. 
Added at VideoOptions enum new value for locking.